app/components/header/header.scss

```diff
@@ -60,39 +51,44 @@ $container-width: 1080px;
 
     .menuItem {
       display: inline-block;
-      margin-right: 20px;
+      margin-right: 30px;
 
       a {
         display: inline-block;
-        width: 72px;
-        font-size: 16px;
+        width: 70px;
+        font-size: 15px;
+        font-weight: 300;
         letter-spacing: 0.5px;
         text-align: center;
-        color: #656d7c;
+        color: #ffffff;
```

Comment:
#ffffff is usually expressed to #fff.
Because white (#ffffff) can be specified with the short notation (#fff) and removes any dependencies on the color depth of the display.